hello_phoenix/ui/lib/ui.ex

```diff
@@ -1,29 +1,9 @@
 defmodule Ui do
```

Comment:
UiWeb?

Reply:
There is _also_ a `UiWeb` module here: https://github.com/nerves-project/nerves_examples/blob/master/hello_phoenix/ui/lib/ui_web.ex